pkg/plugin/clientmgmt/process.go

```diff
@@ -59,7 +61,32 @@ func newProcess(command string, logger logrus.FieldLogger, logLevel logrus.Level
 	// This launches the plugin process.
 	protocolClient, err := client.Client()
 	if err != nil {
-		return nil, err
+		if !strings.Contains(err.Error(), "unknown flag: --features") {
+			return nil, err
+		}
+
+		// Velero started passing the --features flag to plugins in v1.2, however existing plugins
+		// may not support that flag and may not silently ignore unknown flags. The plugin server
+		// code that we make available to plugin authors has since been updated to ignore unknown
+		// flags, but to avoid breaking plugins that haven't updated to that code and don't support
+		// the --features flag, we specifically handle not passing the flag if we can detect that
+		// it's not supported.
+
+		logger.Debug("Plugin process does not support --features flag, removing it and trying again")
```

Comment:
```suggestion
		logger.Debug("Plugin process does not support the --features flag, try removing it and running again")
```

Reply:
hmm I think this should stay as-is -- we aren't asking the user to remove anything, the code itself is removing the `--features` flag and trying again.

Reply:
I'll add the missing `the` though

Reply:
I didn't spot that!

Reply:
added `the`, left the other stuff as-is -- LMK if any more changes are needed!

Reply:
Oh yeah. Duh.